packages/neos-ui/src/Containers/LeftSideBar/NodeTree/Node/index.js

```diff
@@ -1,10 +1,11 @@
 import React, {PureComponent} from 'react';
 import PropTypes from 'prop-types';
 import {$get} from 'plow-js';
+import {compose} from 'redux';
```

Comment:
I like your functional style! But just wondering, do you think it's a good idea to hijack Redux's helper function for this usecase? Maybe we should better use the same method say from lodash (`flowRight`)?
Feels weird to couple html decoding to redux...

Reply:
@dimaip Good catch, yes. Thought so too, but hesitated to bring in another dependency. (`lodash.flowright`) :)

Reply:
I think we can afford it, shouldn't be too fat :)

Reply:
@dimaip I figured that we have the "old" `lodash.compose` in there. Used that instead now.

Reply:
ah perfect